### PR TITLE
Rename Microsoft.Dnx.Compilation.Abstractions to Microsoft.Extensions.Compilation.Abstractions

### PR DESCRIPTION
**Pull request by pakrym**
Issue: https://github.com/aspnet/Universe/issues/318

Also removes unused parameter.
Depends on https://github.com/aspnet/dnx/pull/3126

/cc @pranavkm @davidfowl

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @rynowak \ @dougbu for xunit impact?

File: src/Microsoft.Dnx.TestHost/ProjectCommand.cs [hunk 1]
```diff
@@ -112,13 +109,11 @@ public static bool TryGetEntryPoint(Assembly assembly, out object instance, out
 
             entryPoint = programType.GetRuntimeMethod("TestMain", new[] {typeof (TestHostServices), typeof (string[])});
```
Comment (pranavkm) [inline src/Microsoft.Dnx.TestHost/ProjectCommand.cs]: Ctrl K + D

**Comment by pakrym** [comment]
No impact, that method was always static, added in https://github.com/aspnet/aspnet.xunit/commit/199fc8ba3fc09d6db240e6ae9c905f69eb508090

**Comment by pranavkm** [comment]
Would prefer to get another set of eyes on this before it's checked in.

File: src/Microsoft.Dnx.TestHost/ProjectCommand.cs [hunk 2]
```diff
@@ -110,15 +107,13 @@ public static bool TryGetEntryPoint(Assembly assembly, out object instance, out
...
             }
 
-            entryPoint = programType.GetRuntimeMethod("TestMain", new[] {typeof (TestHostServices), typeof (string[])});
+            entryPoint = programType.GetRuntimeMethod("TestMain", new[] { typeof(TestHostServices), typeof(string[]) });
 
-            if (entryPoint == null)
+            if (entryPoint == null || !entryPoint.IsStatic)
             {
-                System.Console.WriteLine("'{0}' does not contain a 'TestMain' method suitable for an entry point", name);
+                System.Console.WriteLine("'{0}' does not contain static a 'TestMain' method suitable for an entry point", name);
```
Comment (dougbu) [inline src/Microsoft.Dnx.TestHost/ProjectCommand.cs]: "static a" -> "a static"

**Comment by dougbu** [comment]
@pranavkm what was your specific concern? This LGTM and @pakrym should :shipit: to unbreak builds.

**Comment by pranavkm** [comment]
@dougbu no concerns, just FYI since I'm not super familiar with this code

**Pull request merged**